gqlalchemy/query_builder.py

```diff
@@ -74,6 +77,12 @@ class Order(Enum):
     DESCENDING = 4
 
 
+class Operator(Enum):
+    ASSIGNMENT = "="
+    INCREMENT = "+="
+    LABEL_FILTER = ":"
+
+
```

Comment:
It's good that you have created Operator enum, but should we then add all possible operators for comparison here (e.g. >, <,..)? They could be used then by the user when passing an operator to the where clause.

Reply:
Also, should the assignment and label_filter operator then be added in where tests? Also, in where method in query builder _LABEL_FILTER = ":" is used inside the class, rather than from Operator enum. We should maybe fix this accordingly.

Reply:
I will answer myself 😹 . This `Operator` is not the same thing as comparison expressions for `WHERE` clause. Therefore, this is enough for `SET` clause.

Reply:
I would call it `SetOperator` as it's more descriptive.